tests/shared/src/test/scala/cats/tests/NonEmptySeqSuite.scala

```diff
@@ -84,4 +84,10 @@ class NonEmptySeqSuite extends NonEmptyCollectionSuite[Seq, NonEmptySeq, NonEmpt
   test("toNeSeq on empty Seq returns None") {
     assert(Seq.empty[Int].toNeSeq == None)
   }
+
+  test("NonEmptySeq#zip is consistent with Seq#zip") {
+    forAll { (a: NonEmptySeq[Int], b: NonEmptySeq[Int], f: (Int, Int) => Int) =>
```

Comment:
This is unused right?
```suggestion
    forAll { (a: NonEmptySeq[Int], b: NonEmptySeq[Int]) =>
```

Reply:
right,  thanks!